hathor/p2p/sync_v2/agent.py

```diff
@@ -44,6 +44,20 @@
 MAX_GET_TRANSACTIONS_BFS_LEN: int = 8
 
 
+class BlockInfo(NamedTuple):
```

Comment:
Maybe rename to `HeightInfo`?

Reply:
We have a `HeightInfo` class in `hathor.indexes` that is exactly the same. Maybe move that one to some common module and use it in both places? I feel this tuple is so simple that it makes sense for it to be more general.

Reply:
Even though both tuples have the same name `HeightInfo`, they are both used internally and might be independently modified in the future. So I guess we should keep them separate. What do you think?

Reply:
Just renamed it to `_HeightInfo`. Both the commit message and the PR title have been updated.

Reply:
Ok, looks good to me